src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -302,11 +302,17 @@ public List<RESTInformatieobjecttype> listInformatieobjecttypesForZaak(@PathPara
         return informatieobjecttypeConverter.convert(informatieObjectTypes);
     }
 
+    /**
+     * Zet een {@link RESTFileUpload} bestand in de HTTP sessie.
+     *
+     * @param documentReferentieId Zaak-UUID of taak-ID van gerelateerde zaak/taak.
+     * @return Success response
+     */
     @POST
     @Path("informatieobject/upload/{uuid}")
     @Consumes(MediaType.MULTIPART_FORM_DATA)
-    public Response uploadFile(@PathParam("uuid") final UUID uuid, @MultipartForm final RESTFileUpload data) {
-        httpSession.get().setAttribute("FILE_" + uuid, data);
+    public Response uploadFile(@PathParam("uuid") final String documentReferentieId, @MultipartForm final RESTFileUpload data) {
```

Comment:
Hernoem de PathParam en het Path dan ook even naar documentReferentieId, vergeet daarbij dan niet om dit ook te renamen in de frontend.

Reply:
Hernoemd in opvolgende commit